locales/en/LC_MESSAGES/volto.po

```diff
@@ -27,7 +27,7 @@ msgid "Separator"
 msgstr ""
 
 #: components/schema
-# defaultMessage: Separator Block
+# defaultMessage: Separator
```

Comment:
@danalvrz The default message is generated automatically from where the message is defined in code, but I don't see a change to code. Did you update this manually, or did you miss committing the change to the file where the message is defined?

Reply:
I just fixed it, and yes, I got the impression that the change was also for English

Reply:
@danalvrz yep, looks good now